Add "Quick Start" doc. Add "Component Remote State" doc. Add "CLI Commands Cheat Sheet" doc (#280)
cloudposse/atmos: website/docs/cli/commands/terraform/terraform-workspace.md

sidebar_label: workspace
sidebar_class_name: command
id: workspace

> description: This command calculates the `terraform` workspace for an Atmos component (from the context variables and stack config), then runs `terraform init -reconfigure`, then selects the workspace by executing the `terraform workspace select` command.

Reviewer: ```suggestion
description: This command calculates the `terraform` workspace for an Atmos component (from the context variables and stack config), then runs `terraform init -reconfigure` and selects the workspace by executing the `terraform workspace select` command.
```

---

:::note Purpose

> Use this command to calculate the `terraform` workspace for an Atmos component (from the context variables and stack config), then

Reviewer: ```suggestion
Use this command to calculate the `terraform` workspace for an Atmos component (from the context variables and stack config). It will
```

> run `terraform init -reconfigure`, then select the workspace by executing the `terraform workspace select` command.

Reviewer: ```suggestion
run `terraform init -reconfigure` and then select the workspace by executing the `terraform workspace select` command.
```
